### PR TITLE
birthday

### PR DESCRIPTION
**Pull request by azylman**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  }
+
+  set(sid, session, cb) {
+    this.compress(session, (err, compressed) => {
+      this.store.set(sid, compressed, cb);
+    });
+  }
+
+  compress(session, cb) {
+    let uncompressed = JSON.stringify(session);
```
Comment (rgarcia) [inline index.js]: try/catch around JSON.stringify?

also const

File: index.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    this._encoding.compress(uncompressed, (err, compressed) => {
+      if (compressed) {
+        console.log(`turned len ${uncompressed.length} to len ${compressed.length}`)
+      }
```
Comment (rgarcia) [inline index.js:87]: stray log

File: index.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  }
+
+  decompress(session, cb) {
+    this._encoding.decompress(session, (err, uncompressed) => {
+      if (err) {
+        cb(err);
+        return
+      }
+
+      cb(null, JSON.parse(uncompressed));
```
Comment (rgarcia) [inline index.js]: try/catch around JSON.parse?

File: package.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  "main": "index.js",
+  "scripts": {
+    "test": "make test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Clever/connect-session-compress.git"
+  },
+  "author": "azylman",
+  "license": "ISC",
```
Comment (rgarcia) [inline package.json]: MIT plz

File: test/index.js [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+        csc.decompress(compressed, (err, original) => {
+          assert.ifError(err);
+          assert.deepEqual(original, {key: "val"});
+
+          done();
+        });
+      });
+    });
+  });
+});
```
Comment (rgarcia) [inline test/index.js:119]: i think all of the `let`s in this file can be `const`

**Comment by rgarcia** [comment]
made some comments, otherwise lgtm!

File: index.js (hunk 2 above)
Comment (azylman) [inline index.js:87]: @cozmo wanted this in to make sure it was having the expected impact

**Pull request merged**